packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,3 +1,19 @@
+# 53.8.0 - December 16, 2021
```

Comment:
Does this need to be changed to 53.8.1?

Reply:
@randi274 Thats the commit from last weeks release, I didn't update the changelog for the patch.